Apps/Admin/Client/Store/MessageVerification/MessageVerificationReducers.cs

```diff
@@ -68,5 +70,24 @@ public static MessageVerificationState ReduceLoadFailAction(MessageVerificationS
                 ErrorMessage = action.ErrorMessage,
             };
         }
+
+        /// <summary>
+        /// The Reducer for the reset state action.
+        /// </summary>
+        /// <param name="state">The message verification state.</param>
+        /// <returns>The empty state.</returns>
+        [ReducerMethod(typeof(MessageVerificationActions.ResetStateAction))]
+        public static MessageVerificationState ReduceResetStateAction(MessageVerificationState state)
+        {
+            return state with
+            {
+                RequestResult = new()
+                {
+                    ResourcePayload = Enumerable.Empty<MessagingVerificationModel>(),
+                },
```

Comment:
@MikeLyttle  If we use null, it will not show the empty table. Our current admin shows the empty table. We don't follow the existing behavior?

Reply:
That's a separate issue, but yes, the change to that behaviour was deliberate.

The store should have a null RequestResult rather than one with no records because they indicate different things.  A null value indicates that we haven't yet received the result.  We also have a property in the state that determines whether the value has been loaded by comparing it to null.